src/main/kotlin/com/github/patou/gitmoji/GitCommitAction.kt

```diff
@@ -130,16 +130,17 @@ class GitCommitAction : AnAction() {
             if (useUnicode) {
                 var replaced = false
                 for (moji in gitmojis) {
-                    if (message.contains("${moji.emoji} ")) {
-                        message = message.replaceFirst("${moji.emoji} ", "${gitmoji.emoji} ")
+                    if (message.contains("${moji.emoji}${PropertiesComponent.getInstance(project).getValue(CONFIG_AFTER_UNICODE)}")) {
```

Comment:
Extract the config After unicode in a variable will be more performant and lisible